docs/README.md

```diff
@@ -1,29 +1,44 @@
 # Kubernetes Documentation
 
-**Note**
-This documentation is current for HEAD.
+## NOTE: This documentation is current for HEAD.
 
-Documentation for previous releases is available at
-http://releases.k8s.io/vX.Y.Z/docs.
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+If you are using a released version of Kubernetes, you almost certainly want
+the docs that go with that version.
+
+Documentation for specific releases is available at
+releases.k8s.io/v*X*.*Y*.*Z*/docs
+   * [releases.k8s.io/v0.21.0](http://releases.k8s.io/v0.21.0/docs)
```

Comment:
No worries, just curious. We could point it to the releases page on GitHub and fix up the docs links off there.